examples/flask/requirements.txt

```diff
@@ -1,2 +1,35 @@
-Flask==0.10.1
-pagseguro
+arrow==0.12.1
```

Comment:
nitpick: usually we don't need to add all the requirements, like a `pip freeze > requirements.txt`. It can be better just add the specific ones (Flask==1.0.2) and let it add all its dependencies when you run `pip install -r requirements.txt`